operator-common/src/test/java/io/strimzi/operator/common/featuregates/FeatureGatesTest.java

```diff
@@ -54,6 +54,17 @@ public void testFeatureGatesParsing() {
         assertThat(new FeatureGates("+ServerSideApplyPhase1").serverSideApplyPhase1Enabled(), is(true));
         assertThat(new FeatureGates("-ServerSideApplyPhase1").serverSideApplyPhase1Enabled(), is(false));
         assertThat(new FeatureGates("   -ServerSideApplyPhase1   ").serverSideApplyPhase1Enabled(), is(false));
+
+        assertThat(new FeatureGates("+UseConnectBuildWithBuildah").useConnectBuildWithBuildahEnabled(), is(true));
+        assertThat(new FeatureGates("-UseConnectBuildWithBuildah").useConnectBuildWithBuildahEnabled(), is(false));
+        assertThat(new FeatureGates("   -UseConnectBuildWithBuildah   ").useConnectBuildWithBuildahEnabled(), is(false));
+
+        assertThat(new FeatureGates("-ServerSideApplyPhase1,-UseConnectBuildWithBuildah").serverSideApplyPhase1Enabled(), is(false));
+        assertThat(new FeatureGates("-ServerSideApplyPhase1,-UseConnectBuildWithBuildah").useConnectBuildWithBuildahEnabled(), is(false));
+        assertThat(new FeatureGates("  +ServerSideApplyPhase1    ,    +UseConnectBuildWithBuildah").serverSideApplyPhase1Enabled(), is(true));
+        assertThat(new FeatureGates("  +ServerSideApplyPhase1    ,    +UseConnectBuildWithBuildah").useConnectBuildWithBuildahEnabled(), is(true));
+        assertThat(new FeatureGates("+UseConnectBuildWithBuildah,-ServerSideApplyPhase1").serverSideApplyPhase1Enabled(), is(false));
+        assertThat(new FeatureGates("+UseConnectBuildWithBuildah,-ServerSideApplyPhase1").useConnectBuildWithBuildahEnabled(), is(true));
         // TODO: Add more tests with various feature gate combinations once we have multiple feature gates again.
```

Comment:
I guess you can use these now?

Reply:
WDYM? The FGs under the TODO are not usable or do you mean I can remove it?

Reply:
The TODO is about what to do when you have multiple feature gates which we now have. So we should act on it.